apps/em/aat/01/kustomization.yaml

```diff
@@ -3,3 +3,5 @@ kind: Kustomization
 resources:
   - ../base
 namespace: em
+patchesStrategicMerge:
+  - ../../em-hrs-ingestor/schedule-cluster-1.yaml
```

Comment:
can you update naming?

Reply:
They are using across envs to avoid duplication. Will leave it for now.